torch/quantization/ns/graph_passes.py

```diff
@@ -220,6 +220,46 @@ def _insert_dtype_cast_after_node(
     else:
         raise AssertionError(f"type f{type(prev_node_c)} is not handled")
 
+def _copy_node_from_a_to_c(
```

Comment:
I don't have much context, a n00b question, why can't we use node_copy api?

Reply:
thanks, didn't know about that.  Wrote down a TODO to take a look in the near future.  Most likely we can change it, will just need to retest, hoping to do that in a future PR.